src/App.js

```diff
@@ -1,23 +1,14 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import Icon from "./Images/Icon.png";
 
 function App() {
```

Comment:
the function should be named after the Component
in this it should be  something like
```js
function nameOfComponent(){}
export default nameOfComponent;
```